### ISSUE TITLE
display metrics on predicted mask

### ISSUE THREAD

**Issue by joaocarvalho00** [joaocarvalho00/AutomatedMarineGrowth-Classification#15]
when using display_original_mask_predicted_mask function, display the associated stats in the predicted mask

ex:

"predicted mask"
"acc = ...%
"loss = .."
